tests/flow/test_load_csv.py

```
@@ -227,3 +295,62 @@ def test07_load_multiple_files(self):
+        # try accessing files outside of the import directory
+        g = self.graph
+
+        # try accessing the hosts file
+        # default import path: /var/lib/FalkorDB/import/
+        q = """LOAD CSV FROM 'file://../../../etc/hosts AS row
+               RETURN row"""
+
+        try:
+            res = g.query(q).result_set
+            self.env.assertFalse("we should not be here" and False)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)